src/components/Todo/Todo.js

```diff
@@ -1,28 +1,42 @@
 import '../style/Style.css';
-import { useSelector } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
+import { addBook, removeBook } from '../../redux/books/bookSlice';
 
 const MyTodo = () => {
   const { bookData } = useSelector((store) => store.user);
+  const dispatch = useDispatch();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const newBook = {
+      id: formData.get('item_id'),
+      title: formData.get('title'),
+      author: formData.get('author'),
+    };
+    dispatch(addBook(newBook));
+  };
   return (
     <>
       <div className="container">
 
         {bookData.map((item) => (
-          <div key={item.id}>
+          <div key={item.item_id}>
             <h2>{item.title}</h2>
             <p>
               By
               {item.author}
             </p>
-            <button type="button" id="remove">Remove</button>
+            <button type="button" id="remove" onClick={() => dispatch(removeBook())}>Remove</button>
```

Comment:
- [ ] The remove functionality is not working because the redux slice is expecting a payload but you are not passing anything to it. I suggest you pass the book id inside the `removeBook()` function so the deletion can work. 🙂️